src/modules/ekf2/EKF/zero_innovation_heading_update.cpp

```diff
@@ -44,13 +44,12 @@ void Ekf::controlZeroInnovationHeadingUpdate()
 				|| _control_status.flags.ev_yaw || _control_status.flags.gps_yaw;
 
 	// fuse zero innovation at a limited rate if the yaw variance is too large
-	if (_control_status.flags.tilt_align
-	    && !yaw_aiding
+	if(!yaw_aiding
```

Comment:
forgot to mention this in the commit message, but zero heading innovation is also required for the tilt uncertainty to reduce through the fake pos fusion (if the yaw uncertainty is too large, the fake pos fusion has less effect on the tilt variance).